### ISSUE TITLE
Upload items ..

### ISSUE THREAD

**Issue by YafDYaf** [sijeeshmiziha/olx#2]
Hi after updating firebase api key , i've try to upload a post but it stact on the loading part/// any help with that 
please

**Comment by Sion1407** [comment]
Yeah same issue, please reply quick bro I need this project for my final year project!!!!
